cluster/gce/gci/audit_policy_test.go

```diff
@@ -41,7 +41,7 @@ func init() {
 
 func TestCreateMasterAuditPolicy(t *testing.T) {
 	baseDir, err := os.MkdirTemp("", "configure-helper-test") // cleaned up by c.tearDown()
-	require.NoError(t, err, "Failed to create temp directory")
+	require.NoErrorf(t, err, "Failed to create temp directory")
```

Comment:
these changes seem less correct... why is using a constant string with no formatting directives incorrect, and why do we care about preventing it in a linter?